data/json/snippets/newspapers.json

```diff
@@ -0,0 +1,430 @@
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "weeks_old_news",
+    "text": [
+      {
+        "id": "weeks_old_news_1",
+        "text": "SURGEON GENERAL: DEAD ARE STAYING DEAD: \"Stories about the dead rising are completely unfounded and medically impossible.\", the office of the surgeon general said in a press conference today.  \"It's an obvious hoax by people just looking for five minutes of fame, or to scare people in these uncertain times.  After all, remember the clown thing a few years ago?\"  A critic pointed to the increasingly widespread viral videos of rioters with clearly fatal injuries rising to return to the rioting.  \"Within ten hours, someone will have deep faked this video to make me look like a Star Wars character,\" replied the spokesperson, \"and you're confused by a bit of home grown CGI showing zombies?\""
+      },
+      {
+        "id": "weeks_old_news_2",
+        "text": "GENERAL SAYS AVOID BUNKERS: Even in riot affected areas, do not attempt to flee to military bunkers or outposts!  Understaffed bunkers are defended by automated turrets, which will shoot those without proper registration.  FEMA has authorized use of emergency evacuation shelters for people seeking refuge from riots."
+      },
+      {
+        "id": "weeks_old_news_3",
+        "text": "VIOLENCE OUTBREAKS ARE UNIVERSAL.  Despite government efforts to claim the situation is \"under control\", sources in the CDC have confirmed our own investigative results: there are now major unchecked riots in every city in the United States.  Similar reports are trickling in from countries across the globe.  We recommend packing a bugout bag and heading to your cabin on the lake, if you have one."
+      },
+      {
+        "id": "weeks_old_news_4",
+        "text": "ZOMBIE DRUG?  A controversial blogger suggested the recent riots are a chemical attack by China in her latest podcast.  \"They're jealous of our cybernetic superiority, so they teamed up with Haiti to put hoodoo drugs in the water making people zombies!  You've all heard of Forbidden City - who hasn't? - well, this is what they were testing, and now we're seeing it everywhere!\""
+      },
+      {
+        "id": "weeks_old_news_5",
+        "text": "SUPERSOLDIER EXPERIMENTS GONE WRONG.  Recently leaked documents from an inside source in a hitherto unknown government agency abbreivated 'XEDRA' confirm that violent riots across the country are the result of an accidentally released self-replicating supersoldier serum.  The serum gives people unprecedented strength and durability, even regenerative capacity, but has the dangerous side effect of causing uncontrolled anger.  Although the leak suggests that the effect should wear off, it may take weeks or even months to reach a stable state."
```

Comment:
```suggestion
        "text": "SUPERSOLDIER EXPERIMENTS GONE WRONG.  Recently leaked documents from an inside source in a hitherto unknown government agency abbreviated 'XEDRA' confirm that violent riots across the country are the result of an accidentally released self-replicating supersoldier serum.  The serum gives people unprecedented strength and durability, even regenerative capacity, but has the dangerous side effect of causing uncontrolled anger.  Although the leak suggests that the effect should wear off, it may take weeks or even months to reach a stable state."
```